cmake/SetupPackages.cmake

```diff
@@ -81,6 +81,17 @@ if (RAJA_ENABLE_CUDA)
 endif()
 
 
+if (RAJA_ENABLE_TBB)
```

Comment:
@davidbeckingsale will this be changed to 'ENABLE_TBB' for consistency when we switch to BLT?